test/per_file_coverage.sh

```diff
@@ -11,7 +11,7 @@ declare -a KNOWN_LOW_COVERAGE=(
 "source/common/crypto:95.5"
 "source/common/event:95.1" # Emulated edge events guards don't report LCOV
 "source/common/filesystem/posix:96.2" # FileReadToEndNotReadable fails in some env; createPath can't test all failure branches.
-"source/common/http/http2:95.6"
```

Comment:
let's add more tests (even if not of this code) rather than simply decrease coverage.

Reply:
Done.